apis/reaper/v1alpha1/reaper_types.go

```diff
@@ -155,6 +136,30 @@ type AutoScheduling struct {
 	ExcludedKeyspaces []string `json:"excludedKeyspaces,omitempty"`
 }
 
+type ReaperClusterTemplate struct {
+	ReaperDatacenterTemplate `json:",inline"`
+
+	// The keyspace to use to store Reaper's state. Will default to "reaper_db" if unspecified. Will be created if it
+	// does not exist, and if this Reaper resource is managed by K8ssandra.
+	// +kubebuilder:default="reaper_db"
+	// +optional
+	Keyspace string `json:"keyspace,omitempty"`
+
+	// Defines the username and password that Reaper will use to authenticate CQL connections to Cassandra
+	// clusters. These credentials need to be stored on each Cassandra node. If CQL authentication is not required,
+	// leave this field empty. The secret must be in the same namespace as Reaper itself and must contain two keys:
+	// "username" and "password".
+	// +optional
+	CassandraUserSecretRef string `json:"cassandraUserSecretRef,omitempty"`
+
+	// Defines the username and password that Reaper will use to authenticate JMX connections to Cassandra
+	// clusters. These credentials need to be stored on each Cassandra node. If JMX authentication is not required,
```

Comment:
Both actually. The jmx credentials will be stored on Cassandra nodes in a `jmxremote.password` file; on Reaper nodes, it will be stored in `cassandra-reaper.yml`. I will fix the comment here too.